eventhubs/data-plane/azure-eventhubs/src/main/java/com/microsoft/azure/eventhubs/impl/ReplayableWorkItem.java

```diff
@@ -32,7 +33,7 @@ private void initialize(final byte[] amqpMessage, final int encodedMessageSize,
     }
 
     public byte[] getMessage() {
-        return this.amqpMessage;
+        return this.amqpMessage != null ? Arrays.copyOf(this.amqpMessage, this.amqpMessage.length) : new byte[0];
```

Comment:
You can't do this

Reply:
Do you mean I can't use the ternary operator or I should't change the code or something else?

Reply:
You shouldn't change this code.